soweego/linker/link.py

```diff
@@ -463,25 +277,36 @@ def _upload(chunk, chunk_number, catalog, entity, sandbox):
 
 def _add_missing_feature_columns(classifier, feature_vectors: pd.DataFrame):
     # Handle amount of features depending on the classifier
+    expected_features: int
```

Comment:
Can you please explain why you need this?

Reply:
Just to make it clearer that `expected_features` is an `int`. Do you think we should remove it?

Reply:
No, that's fair enough.